### PR TITLE
ci(test): Fix nmi postman collection

### PR DESCRIPTION
**Pull request by Gnanasundari24**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description
 Save card flows are failing in nmi collection. Fixed that in this PR

## Motivation and Context

fix

## How did you test it?
Screenshot 2024-09-17 at 9 07 20 PM

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Comment by pixincreate** [comment]
can you check the checks failing?

File: Cargo.lock
Comment (pixincreate) [inline Cargo.lock:1]: can you revert this change?

**Review by pixincreate — APPROVED** [review]

**Review by likhinbopanna — APPROVED** [review]

**Review by likhinbopanna — APPROVED** [review]

**Review by pixincreate — APPROVED** [review]
